pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -1398,7 +1398,7 @@ class ShapePropagator : public PropertyPropBase {
             if (maybe_dtype_option && !maybe_dtype_option->isNone()) {
               return {ret->withScalarType(maybe_dtype_option->toScalarType())};
             } else {
-              return {ret};
+              return {};
```

Comment:
Why not return the input type as it's?

Reply:
OK, return null to skip some inputs like prim::dtype. I have dealt with prim::dtype now.